tests/interpolation/built_in_resolvers/test_dict.py

```diff
@@ -234,33 +234,18 @@ def test_dict_values_dictconfig_resolver_output(
     assert cfg.foo[key] == expected
 
 
-@mark.parametrize(
-    ("make_resolver", "expected_value", "expected_content"),
-    [
-        param(
-            lambda _parent_: OmegaConf.create({"a": 0, "b": 1}, parent=_parent_),
-            [0, 1],
-            ["${y.a}", "${y.b}"],
-            id="dictconfig_with_parent",
-        ),
-        param(
-            lambda: {"a": 0, "b": 1},
-            [0, 1],
-            ["${y.a}", "${y.b}"],
-            id="plain_dict",
-        ),
-    ],
-)
-def test_dict_values_transient_interpolation(
+@mark.parametrize("dict_func", ["oc.dict.values", "oc.dict.keys"])
+def test_extract_from_dict_resolver_output(
```

Comment:
I don't understand what this function is testing from its name.
Is this new test related to the changes in this PR?

Reply:
The name means "test what happens when you try to extract [keys / values] from a dictionary which is the output of a resolver".

The relation with this PR is that before this commit, in this test `y` would be a DictConfig and thus you would be able to do `${oc.dict.keys:y}` or `${oc.dict.values:y}`.
But now `y` is a plain Python dictionary which isn't supported by `oc.dict.{keys,values}`.

This is essentially the replacement for the previous test `test_dict_values_transient_interpolation[plain_dict]` (which was poorly named because the interpolation isn't transient anymore, it was legacy from an earlier version)

The other previous test `test_dict_values_transient_interpolation[dictconfig_with_parent]` is gone because it seemed too redundant with `test_dict_values_dictconfig_resolver_output[basic]` to be worth keeping it.